docs/src/domains/steps/_index.md

```diff
@@ -25,7 +25,7 @@ However, the domain used for non-production environments will always be generate
 even if your project is on a Production plan.
 {{< /note >}}
 
-Go to [your account](https://accounts.platform.sh/user), click on the small wheel next to you project's name and click on edit.
+In the [console](https://console.platform.sh), click the **More** button for your project and select **Edit plan**.
```

Comment:
I assume that it's named "More" because of its `aria-label`, as per:
https://developers.google.com/style/ui-elements?hl=en#buttons
?

Reply:
Yep. This was just for typos and there are more things on the page that are wrong, so I didn't bother adding the SVG and fixing the other things just now.